### ISSUE TITLE
[LIB REQUEST] INSERT LIB NAME HERE

### ISSUE THREAD

**Issue by nico-liao0918** [compiler-explorer/compiler-explorer#6817]
Harmony

**Comment by dkm** [comment]
It would probably help a bit more if you gave us more into.
Do you mean https://github.com/pardeike/Harmony ?

**Comment by nico-liao0918** [comment]
Yes. I mean https://github.com/pardeike/Harmony. This Harmony to be added. Sorry for the misinformation and misunderstanding.
